Tabloid/client/src/components/TagList.js

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState } from "react";
+import { getAllTags } from "../modules/tagManager";
+
+const TagList = () => {
+  const [tags, setTags] = useState([]);
+
+  const getTags = () => {
+    getAllTags().then(tags => setTags(tags));
+  };
+
+  useEffect(() => {
+    getTags();
+  }, []);
+
+  return (
+    <div>
+      {tags.map(t => 
+        <div>{t.name}</div>
```

Comment:
This isn't using the Tag method from Tag.js and is instead just displaying the tag manually. If you'd like to do that, that's fine, but we can't have both. Either use the Tag component to display them or remove it, please.